app/helpers/application_helper/toolbar/diagnostics_server_center.rb

```diff
@@ -79,12 +79,6 @@ class ApplicationHelper::Toolbar::DiagnosticsServerCenter < ApplicationHelper::T
         ),
       ]
     ),
-    button(
-      :log_depot_edit,
-      'pficon pficon-edit fa-lg',
-      N_('Edit the Log Depot settings for the selected Server'),
-      N_('Edit'),
-      :klass => ApplicationHelper::Button::LogDepotEdit),
```

Comment:
Looks like these can all go away now too?

```
10:14:18 ~/dev/manageiq-ui-classic (fix_cypress) (2.7.4) ✓ git grep LogDepotEdit
app/helpers/application_helper/button/collect_logs.rb:1:class ApplicationHelper::Button::CollectLogs < ApplicationHelper::Button::LogDepotEdit
app/helpers/application_helper/button/log_depot_edit.rb:1:class ApplicationHelper::Button::LogDepotEdit < ApplicationHelper::Button::Basic
app/helpers/application_helper/button/zone_collect_logs.rb:1:class ApplicationHelper::Button::ZoneCollectLogs < ApplicationHelper::Button::LogDepotEdit
app/helpers/application_helper/toolbar/diagnostics_server_center.rb:87:      :klass => ApplicationHelper::Button::LogDepotEdit),
app/helpers/application_helper/toolbar/diagnostics_zone_center.rb:95:      :klass => ApplicationHelper::Button::LogDepotEdit),
spec/helpers/application_helper/buttons/log_depot_edit_spec.rb:1:describe ApplicationHelper::Button::LogDepotEdit do
```

Reply:
Yup yup! I'm already on it